.github/workflows/build_macos.yaml

```diff
@@ -9,14 +9,19 @@ jobs:
       matrix:
         config: [debug, release]
         compiler: [clang]
-        arch: [x86, arm64]
+        os: [13, 14, 15]
+        # arch: [x86, arm64]]
```

Comment:
You could get arch from `uname -m`. I've tested it [here](https://github.com/tothxa/widelands/pull/22/files). OK, it won't be available for the matrix job name, where it's not very important IMO, but you can remove the `include` block, which is a big clean-up IMO.